packages/checkout/components/store-notices-container/store-notices.tsx

```diff
@@ -17,11 +17,11 @@ import { getClassNameFromStatus } from './utils';
 import type { StoreNotice } from './types';
 
 const StoreNotices = ( {
-	context,
+	contexts,
```

Comment:
You'll need to update the docs [here](https://github.com/woocommerce/woocommerce-blocks/blob/try/shipping-captures-billing-notices/docs/internal-developers/block-client-apis/notices.md?plain=1#L26) as well to reflect plural